### PR TITLE
Switch to image.sc support forum by default

### PR DESCRIPTION
**Pull request by normanrz**
Switches the support links to https://forum.image.sc/tag/webknossos

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Updated [documentation](../blob/master/docs) if applicable
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/navbar.js [hunk 1]
```diff
@@ -300,6 +300,18 @@ function LoggedInAvatar({ activeUser, handleLogout, ...other }) {
         <Menu.Item disabled key="organization">
           {orgDisplayName}
         </Menu.Item>
+        <Menu.Item key="switch">
```
Comment (fm3) [inline frontend/javascripts/navbar.js:303]: This does not really seem to work for me. When I click it (in Firefox) the menu closes, so I can’t type in it to see what it does. Also, this should probably only appear in the isMultiMember case, right?
From the code, I don’t really understand what it is supposed to do.

Comment (normanrz) [reply frontend/javascripts/navbar.js:303]: This was not supposed to be part of the PR. Just a failed experiment. Would be nice to have something like this, tho :D

**Review by philippotto — COMMENTED** [review]
Looks good to me :+1: However, the autocomplete-switch-stuff is still in the PR.

**Review by philippotto — APPROVED** [review]
:+1:

**Pull request merged**